config/initializers/new_framework_defaults_6_0.rb

```diff
@@ -37,9 +37,9 @@
 # If you send mail in the background, job workers need to have a copy of
 # MailDeliveryJob to ensure all delivery jobs are processed properly.
 # Make sure your entire app is migrated and stable on 6.0 before using this setting.
-# Rails.application.config.action_mailer.delivery_job = "ActionMailer::MailDeliveryJob"
+Rails.application.config.action_mailer.delivery_job = "ActionMailer::MailDeliveryJob"
```

Comment:
Just reading the comment above this one, is it worth holding off on flipping this until after this PR is merged and deployed?

Reply:
oops didn't spot this. Yes!